### PR TITLE
Script building all of the extra RPM packages

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

File: bin/build_extra_package.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+require 'manageiq/rpm_build/extra_package'
+
+Dir.chdir(ManageIQ::RPMBuild::ExtraPackage::PACKAGES_DIR) do
+  directories = (Dir["*"] - ["."]).reject { |o| !File.directory?(o) }
```
Comment (Fryguy) [inline bin/build_extra_package.rb:7]: Minor, but for consistency, this should probably be sorted

File: bin/build_extra_package.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+require 'manageiq/rpm_build/extra_package'
+
+Dir.chdir(ManageIQ::RPMBuild::ExtraPackage::PACKAGES_DIR) do
```
Comment (Fryguy) [inline bin/build_extra_package.rb:6]: This chdir feels unnecessary if it's only used for the Dir glob.

File: bin/build_extra_package.rb (hunk 1 above)
Comment (bdunne) [reply bin/build_extra_package.rb]: Done

File: bin/build_extra_package.rb (hunk 2 above)
Comment (bdunne) [reply bin/build_extra_package.rb:6]: If I don't chdir, the directories will be fully qualified paths which will require further processing.

File: lib/manageiq/rpm_build/extra_package.rb [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+          "kafka"       => {
+            :mock_extras => "--enable-network",
+            :os          => "centos-stream+epel",
+          },
+          "qpid-proton" => {
+            :os => "centos-stream+epel",
+          },
+          "repmgr13"    => {
+            :os => "centos-stream+epel",
+          },
```
Comment (Fryguy) [inline lib/manageiq/rpm_build/extra_package.rb:92]: looks like you will need some extra config based on #341

**Pull request merged**